src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -702,7 +702,7 @@ std::pair<bool, Uptane::Target> SotaUptaneClient::downloadImage(const Uptane::Ta
 
   // Note: handle exceptions from here so that we can send reports and
   // DownloadTargetComple events in all cases. We might want to move these to
-  // downloadImages but aktualizr-lite currently calls this method directly.
+  // downloadImages
```

Comment:
Is the fragment you removed no longer true?

Reply:
good catch. i've force pushed a fix.